scripts/liveMonitor_countTests/count-java-tests.py

```diff
@@ -0,0 +1,84 @@
+from sys import argv
+import os
+import glob
+
+if len(argv) != 2:
+    print("Usage: count-java-tests.py [directory]")
+    exit()
+
+searchDir = argv[1]
+if not os.path.isdir(searchDir):
+    print("Error! '" + searchDir + "' is not a directory")
+    exit()
+
+print("\nCounting tests in '" + searchDir + "' ...")
+
+def get_filepaths(directory, extension):
+    file_paths = []
+    for root, directories, files in os.walk(directory):
+        for filename in files:
+            if filename.endswith(extension):
+                filepath = os.path.join(root, filename)
+                file_paths.append((root, filepath, filename))
+    return file_paths
+
+# Run the above function and store its results in a variable.   
+full_file_paths = get_filepaths(searchDir, "java")
+
+print("\nFound " + str(len(full_file_paths)) + " java files\n")
+
+searchDirParts = searchDir.split('/')
+
+# count per directory and running count
+count = 0
+paths = {}
+for root, filepath, filename in full_file_paths:
+    fopen = open(filepath,mode='r')
+    found = False
+    fread = fopen.readlines()
+    for line in fread:
+        if '@test' in line:
+            found = True
+            break
```

Comment:
Since we have a break here, this count will only count 1 even if the file has multiple tests defined by `@test`. We can probably enhance the count by count the number of tests inside one java file. This could be a future work.